composer.json

```diff
@@ -5,7 +5,7 @@
     "require": {
         "php": ">=7.4",
         "magento/framework": "*",
-        "aligent/async-events": "^2.3"
+        "aligent/async-events": ">=2"
```

Comment:
I'd probably prefer a blanket `>=` is not used, just to avoid potential compatibility issues in the future.
If it's compatible with v3, then can use either:
 - `>= 2 < 4`
 - `^2.3|^3.0`

Reply:
`^2.3|^3.0` Sounds good. I guess the release after that can be `^3.0` only